src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/CoverageVisualisationService.kt

```diff
@@ -71,15 +75,37 @@ class CoverageVisualisationService(private val project: Project) {
             relativeMutations = (coveredMutations.toDouble() / allMutations * 100).roundToInt()
         }
 
-        visualisationService.panelTitleAbsolute.text = "Absolute Test Coverage for ${testReport.UUT}"
-        visualisationService.panelTitleRelative.text = "Relative Test Coverage for ${testReport.UUT}"
+        // Change the values in the table
+        visualisationService.data[4] = testReport.UUT
+        visualisationService.data[5] = "$relativeLines% ($coveredLines/$allLines)"
+        visualisationService.data[6] = "$relativeBranch% ($coveredBranches/$allBranches)"
+        visualisationService.data[7] = "$relativeMutations% ($coveredMutations/ $allMutations)"
```

Comment:
It's a bit confusing to have a `CoverageToolWindowDisplayService` named `visualisationService`, inside the `CoverageVisualisationService`. I get what you mean, but the name of that variable is misleading.

Reply:
Fixed!